app/Http/Kernel.php

```diff
@@ -43,6 +43,9 @@ class Kernel extends HttpKernel
             'throttle:60,1',
             'bindings',
         ],
+		'platform' => [
+			LocaleMiddleware::class
```

Comment:
This is unnecessary - we are not using the platform middleware group at all, also the locale middleware is on the web middleware group that is applied to the text platform already